go/vt/tlstest/tlstest_test.go

```diff
@@ -238,3 +269,50 @@ func testConfigGeneration(t *testing.T, rootPrefix string, generateConfig func(C
 	}
 
 }
+
+func getCertPayload(pathes ... string) ([]byte, error) {
```

Comment:
I think by mistake left a comment on the commit itself earlier, I think this should be `paths`.

Reply:
removed this method completely, end up with another set of tests.